src/main/java/com/github/wovnio/wovnjava/CustomDomainLanguageValidator.java

```diff
@@ -0,0 +1,39 @@
+package com.github.wovnio.wovnjava;
+
+import java.util.ArrayList;
+import java.net.URL;
+
+class CustomDomainLanguageValidator {
+    private CustomDomainLanguageValidator() {}
+
+    public static ValidationResult validate(ArrayList<CustomDomainLanguage> customDomainLanguageList, ArrayList<Lang> supportedLangs) {
+        ArrayList<Lang> includedLangs = new ArrayList<Lang>();
+        ArrayList<String> includedUrls = new ArrayList<String>();
+
+        for (CustomDomainLanguage customDomainLanguage : customDomainLanguageList) {
```

Comment:
I think there's a `groupingBy` stream collector that could express this more concisely, but this is fine as well.